integration-tests/dynamic-config-tests/src/test/java/org/apache/dynamicconfig/test/DynamicConfigurationIT.java

```diff
@@ -0,0 +1,144 @@
+    Map<String,String> body = new HashMap<>();
+    body.put("releaseTitle", "release-configuration");
+    body.put("releasedBy", "apollo");
+    entity = new HttpEntity<Object>(body, headers);
+    exchange = rest.exchange(release, HttpMethod.POST, entity, String.class);
+
+  }
+
+  @Test
+  public void testDynamicConfiguration() {
+    //before
+    Assert.assertEquals(DynamicPropertyFactory.getInstance().getStringProperty("loadbalcance", "default").getValue(), "default");
+
+    //set and return 200. release 200,update return 200
+
+    String setLoadBalance = url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/items";
+
+    String release = url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/releases";
+    RestTemplate rest = new RestTemplate();
+    HttpHeaders headers = new HttpHeaders();
+    headers.add("Content-Type", "application/json;charset=UTF-8");
+    headers.add("Authorization", token);
+
+    //set new configuration item
+    Map<String, String> body = new HashMap<>();
+    body.put("key", "loadbalance");
+    body.put("value", "roundrobbin");
+    body.put("dataChangeCreatedBy", "apollo");
+    HttpEntity<?> entity = new HttpEntity<Object>(body, headers);
+
+    ResponseEntity<String> exchange = rest.exchange(setLoadBalance, HttpMethod.POST, entity, String.class);
+    Assert.assertEquals(exchange.getStatusCodeValue(), HttpStatus.OK.value());
+
+    //relese loadbalance item
+    body.clear();
+    body.put("releaseTitle", "release-configuration");
+    body.put("releasedBy", "apollo");
+    entity = new HttpEntity<Object>(body, headers);
+    exchange = rest.exchange(release, HttpMethod.POST, entity, String.class);
+    Assert.assertEquals(exchange.getStatusCodeValue(), HttpStatus.OK.value());
+
+    //waiting for a refresh cycle
+    await().atMost(5, SECONDS).until(
+        () -> DynamicPropertyFactory.getInstance().getStringProperty("loadbalance", "default").getValue()
+            .equals("roundrobbin"));
+
+    //update loadbalance value
+    String updateLoadBalance =
+        url + "/openapi/v1/envs/DEV/apps/SampleApp/clusters/default/namespaces/application/items/" + "loadbalance";
+    body.clear();
+    body.put("key", "loadbalance");
+    body.put("value", "random");
+    body.put("dataChangeLastModifiedBy", "apollo");
+    entity = new HttpEntity<Object>(body, headers);
+    exchange = rest.exchange(updateLoadBalance, HttpMethod.PUT, entity, String.class);
+    Assert.assertEquals(exchange.getStatusCodeValue(), HttpStatus.OK.value());
+
+    //release again
+    body.clear();
+    body.put("releaseTitle", "test-release");
+    body.put("releasedBy", "apollo");
+    entity = new HttpEntity<Object>(body, headers);
+    exchange = rest.exchange(release, HttpMethod.POST, entity, String.class);
+    Assert.assertEquals(exchange.getStatusCodeValue(), HttpStatus.OK.value());
```

Comment:
The release logic can extract to be a method. Besides, I recommend to test each operation in a standalone test as it's easier to figure out which part goes wrong by the test method name when it fails.

Reply:
That make sense.